tests/libtest.sh

```diff
@@ -29,6 +29,14 @@ else
   test_builddir=$(dirname $0)
 fi
 
+# We need to get the real path in order to use valgrind
+if test -n "${OT_TESTS_UNINSTALLED:-}"; then
```

Comment:
@cgwalters: See this commit to get the idea. Seems to work.

https://github.com/kinvolk/ostree/commit/5700c7705f89119578a0e882343dca2e0bf58e15